json/komga.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Komga",
+    "slug": "komga",
+    "categories": [
+        12
+    ],
+    "date_created": "2024-11-15",
+    "type": "ct",
+    "updateable": true,
+    "privileged": false,
+    "interface_port": "25600",
+    "documentation": null,
+    "website": "https://komga.org/",
+    "logo": "https://github.com/gotson/komga/raw/master/.github/readme-images/app-icon.png",

```

Comment:
This icon has a white background, have you tested how it looks on the website?

Reply:
nvm I can see it in the readme, guess it's just the browser adding white background when I open it